src/promptflow/promptflow/_core/tool_meta_generator.py

```diff
@@ -269,12 +270,12 @@ def generate_tool_meta_dict_by_file(path: str, tool_type: ToolType):
     note that if a python file is passed, correct working directory must be set and should be added to sys.path.
     """
     tool_type = ToolType(tool_type)
-    file = Path(path).resolve()
+    file = Path(path).resolve(strict=False)
```

Comment:
Why change strict=False?

Reply:
The original issue was Path(path).resolve() raised `FileNotFoundError`.
Which will only happen when `strict=True`.
Although didn't find any python version's default strict value is True, manually set it as False to avoid future issue.

Reply:
We may add comments here to explain this.

Reply:
After re-consider the implementation, the `resolve` should not be necessary there.
Just remove it.